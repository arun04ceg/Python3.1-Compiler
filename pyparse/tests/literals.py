'\100'
'\x40'
'\500'
'\x7E'
'\x09'
'\176'
'\000'
'\x61\\r\n'

"""this
is\
a
string\n."""
r"""this
is\
a
string\n."""

3.14
10.
.001
1e100
3.14e-10
0e0

b'\x7f\x45\x4c\x46\x01\x01\x01\x00'
rb"\x7fELF\x01\x01\x01\0"
B'''\x7f'ELF'\x01\x01\x01\0''latin-1'''
bR"""7f454c4601010100'"hex"""
Rb'\xc3\x9feta'
RB"Repetition of re sequence 1 or more times."
rb'\xc3\x9f\x65\x74\x61'
b'\xc3\x83\xc2\x9feta\''

