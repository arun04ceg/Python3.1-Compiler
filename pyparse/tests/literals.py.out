(program
 (expr "@")
 (expr "@")
 (expr "@")
 (expr "~")
 (expr "\t")
 (expr "~")
 (expr "")
 (expr "a\\r\n")
 (expr "this\nisa\nstring\n.")
 (expr "this\nis\\\na\nstring\\n.")
 (expr 3.14)
 (expr 10.0)
 (expr 0.001)
 (expr 1e+100)
 (expr 3.14e-10)
 (expr 0.0)
 (expr "\u007FELF\u0001\u0001\u0001")
 (expr "\\x7fELF\\x01\\x01\\x01\\0")
 (expr "\u007F'ELF'\u0001\u0001\u0001")
 (expr "7f454c4601010100'\"hex")
 (expr "\\xc3\\x9feta")
 (expr "Repetition of re sequence 1 or more times.")
 (expr "\\xc3\\x9f\\x65\\x74\\x61")
 (expr "Ãƒ\u009Feta'"))

